src/model.jl

```diff
@@ -215,6 +256,16 @@ function _build_model(m::JuMP.Model; case::Case, storage::Storage,
         storage_map = sparse(storage_bus_idx, sets.storage_idx, 1,
                              sets.num_bus, sets.num_storage)::SparseMatrixCSC
     end
+    # Demand flexibility parameters (if present)
+    bus_flex_amt = _make_bus_flexibility_amount(
+        case, flexibility, start_index, end_index
+    )
+    flexibility_enabled = (bus_flex_amt != zeros(sets.num_bus, interval_length))
```

Comment:
Yeah I think that is a great idea. For the first `flexibility_enabled` idea, I thought it wasn't necessary because of the way the Scenario framework worked (or at least how I currently understand it). If someone was going to run comparisons, wouldn't that show up as separate scenarios? I may very well be wrong on that. Regardless, what you suggested would be great for people who don't use/aren't currently using the Scenario framework and are just using input data with `REISE.jl`. I'm guessing something like this would make sense to be included as a command line input, like what is enabled through `pyreisejl/utility/call.py`?

For the second `flexibility_enabled` idea, I think that is great too. Certainly relevant when considering the rollout/availability of demand flexibility to different parts of the country. That would probably have to be some sort of profile input, likely mapping demand flexibility availability to different load zones (or similar defined areas). I can certainly look into doing that. Would you like that included in this PR?

Reply:
I want to follow-up on these two suggestions. For the first (including an overarching `demand_flexibility_enabled` that dictates whether or not demand flexibility is enabled, even if the demand flexibility profile is included), I've been thinking about how something like this would be implemented. There are a couple of options that I can think of: (1) including a command line input, similar to what is enabled through `pyreisejl/utility/call.py` and (2) including a `demand_flexibility_enabled` parameter in `demand_flexibility_parameters.csv`, which is a file I am currently using to specify the duration parameter and was thinking about using to specify which combination of the balancing constraints should be enforced. If we were to implement (1), we would probably want to implement this for other resources too, right? I imagine something similar could be done for energy storage. My concerns with this is that (a) there could be a lot of inputs if we do this for a lot of resources and (b) this would only work when using the Python implementation. For (2), the implementation would probably be a little clunky, but functional. Any time a user wanted to change their preference on allowing demand flexibility, they would need to open and update a .csv file (or generate a new scenario). I would love to hear your thoughts on these options (and potentially other ideas).

For the second suggestion (being able to specify demand flexibility participation by region), I was thinking that should be saved for a future PR. I am planning to create issues for the ideas that have been discussed in this PR and offline that will be saved for future PRs. This should help us keep track of some of these desired features.

Reply:
Thanks @lanesmith, that's good summary of what we have been discussed last week. I think we need inputs from @rouille @danielolsen at this point.

Reply:
I added `demand_flexiblity_enabled` using (1) as I suggested above (the method where the parameter is contained within `demand_flexibility_parameters.csv`); if desired, this can be changed. I also added the ability for the user to specify (again via `demand_flexibility_parameters.csv`) the combination of interval load balance and rolling load balance constraints they would like included in the model.